### PR TITLE
_ update ko_KR translation

### PR DESCRIPTION
**Pull request by epsimatt**
@jbarrett _

### CODE REVIEW

**Comment by epsimatt** [comment]
@jbarrett just finished updating my branch to resolve all conflicts

File: locales/ko_KR/LC_MESSAGES/duckduckgo.po [hunk 1]
```diff
@@ -73,15 +73,15 @@ msgctxt "noresults"
 msgid ""
 "%sClick here%s to try again, if you think there should be results for this "
 "search."
-msgstr ""
+msgstr "%여기%를 클릭해서 다시 시도해 보세요."
```
Comment (jbarrett) [inline locales/ko_KR/LC_MESSAGES/duckduckgo.po]: This needs `%s` format placeholders - is this correct?

```suggestion
msgstr "%s여기%s를 클릭해서 다시 시도해 보세요."
```

**Comment by jbarrett** [comment]
Very sorry for the delay @epsimatt - many thanks for this!

I just saw one issue...

File: locales/ko_KR/LC_MESSAGES/duckduckgo.po (hunk 1 above)
Comment (epsimatt) [reply locales/ko_KR/LC_MESSAGES/duckduckgo.po]: _ fixed in https://github.com/duckduckgo/duckduckgo-locales/pull/34/commits/94ba72613d1aa79fc621421aa2185843584f1785. thankies!

**Comment by jbarrett** [comment]
Thank you very much @epsimatt!

**Pull request merged**